Jenkinsfile

```diff
@@ -126,7 +126,9 @@ pipeline {
       runbld(stashedTestReports: stashedTestReports, project: env.REPO)
     }
     cleanup {
-      notifyBuildResult(prComment: true, slackComment: true, slackNotify: (isBranch() || isTag()))
+      notifyBuildResult(prComment: true,
+                        slackComment: true, slackNotify: (isBranch() || isTag())
+                        analyzeFlakey: true, flakyReportIdx: "reporter-beats-pipeline-master")
```

Comment:
I don't know if we should use as many indices as release branches we got and use the `CHANGE_TARGET` for the PRs or BRANCH_NAME for the branches/tags.

But, anytime there is a new release, then we might need to create a new indice.

@cachedout , what are your thoughts?

Reply:
Yeah, this is an issue we need to figure out. I think that the thing to do here is to use one index per project and then include a field that we can filter against when searching for documents, but to achieve that I think we need to make some changes to the way we actually generate the flaky test analysis.

For now, I propose that we keep it as-is and address this in the test analyzer itself. Then we can come back and change the pipelines as-needed.